assets/js/swedbank-pay-design-guide-theme.js

```diff
@@ -70,8 +72,8 @@
 })();
 
 // Initialize Tipue search
-(function() {
-    $(document).ready(function() {
-        $('#tipue_search_input').tipuesearch();
+(function () {

```

Comment:
Ugh! Please get rid of this space somehow.

```suggestion
(function() {
```